### PR TITLE
add new endpoint to cleanup all obsolete repo's with 404 error codes

### PR DESCRIPTION
**Pull request by danielyhuang**

### CODE REVIEW

File: src/main/java/com/capitalone/dashboard/controller/GitHubController.java [hunk 1]
```diff
@@ -60,4 +60,20 @@ public GitHubController(BaseCollectorRepository<Collector> collectorRepository,
                         + " Records updated: " + collector.getLastExecutionRecordCount());
     }
 
+
+    @RequestMapping(value = "/cleanup", method = GET, produces = APPLICATION_JSON_VALUE)
+    public ResponseEntity<String> cleanup() throws HygieiaException {
+        Collector collector = collectorRepository.findByName(GITHUB_COLLECTOR_NAME);
+        if (Objects.isNull(collector))
+            return ResponseEntity.status(HttpStatus.OK).body(GITHUB_COLLECTOR_NAME + " collector is not found");
```
Comment (rvema) [inline src/main/java/com/capitalone/dashboard/controller/GitHubController.java:68]: not a good practise to have logic in the controller, we would need to create a service layer and implement the DAO logic at the service layer.

**Review by rvema — CHANGES_REQUESTED** [review]
Please address the review comments.

File: src/main/java/com/capitalone/dashboard/controller/GitHubController.java (hunk 1 above)
Comment (danielyhuang) [reply src/main/java/com/capitalone/dashboard/controller/GitHubController.java]: completed as suggested

**Review by rvema — APPROVED** [review]
lgtm

**Review by nireeshT — APPROVED** [review]
LGTM

**Pull request merged**